Exercise_A/assets/css/style.css

```diff
@@ -1,3 +1,195 @@
 /* 
 Add in your custom style such that you can develop the wonderful website within the exercise. 
 */
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Source Sans Pro', sans-serif;
+}
+
+.header {
+    min-height: 100vh;
+    width: 100%;
+    background-image: linear-gradient(rgba(59, 50, 50, 0.7), rgba(63, 53, 53, 0.7)), url(../images/header.jpg);
+    background-position: center;
+    background-size: cover;
+    position: relative;
+}
+
+
+
+.home {
+    width: 90%;
+    color: lightgray;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+}
+
+.home h1 {
+    font-size: 70px;
+    color: white;
+}
+
+.home p {
+    margin: 10px 0 40px;
+    font-size: 20px;
+    color: lightgray;
+}
+
+
+
+.first-line {
+    color:red;
+    opacity: 1;
+    width: 50px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-bottom: 3px;
+}
+
+#navColor{
+    color: lightgray;
+}
+
+#navLine{
+    border-bottom: grey 1px solid;
+}
+
+#page2hr {
+    color:white;
+    opacity: 1;
+    width: 50px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-bottom: 3px;
+}
+
+
+
+.page2 {
+    width: 100%;
+    margin: auto;
+    text-align: center;
+    background: rgb(240, 95, 64);
+    color: white;
+}
+
+.page2Text{
+    padding-top: 70px;
+    padding-bottom: 70px;
+}
+.page3 {
+    width: 100%;
+    margin: auto;
+    text-align: center;
+    background: white;
+    color: black;
+}
+
+.page3Text{
+    padding-top: 70px;
+    padding-bottom: 70px;
+}
+
+#page3hr {
+    color:red;
+    opacity: 1;
+    width: 50px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-bottom: 3px;
+}
+
+#globeIcon{
+    font-size: 50px;
+    color: rgb(240, 95, 64);
+    padding-bottom: 20px ;
+}
+#planeIcon{
+    font-size: 50px;
+    color: rgb(240, 95, 64);
+    padding-bottom: 20px ;
+}
+#likeIcon{
+    font-size: 50px;
+    color: rgb(240, 95, 64);
+    padding-bottom: 20px ;
+}
+#heartIcon{
+    font-size: 50px;
+    color: rgb(240, 95, 64);
+    padding-bottom: 20px ;
+}
+

```

Comment:
The ids applied to the icons that you have placed above this comment repeat way too much code.
Make a single class (icon), you can apply this class to all of the elements that contain you icons.

    font-size: 50px;
    color: rgb(240, 95, 64);
    padding-bottom: 20px ;

is what is in the class. You have repeated this code 3 times. This is bad coding.

Try to stick to one colour convention.